configure.ac

```diff
@@ -927,6 +927,13 @@ else
   ZEND_DEBUG=no
 fi
 
+PHP_ARG_ENABLE(rtld-now, whether to dlopen extensions with RTLD_NOW instead of RTLD_AZY,
+[  --enable-rtld-now       Use dlopen with RTLD_NOW instead of RTLD_LAZY], no, no)
```

Comment:
This can be "simplified"/complicated with AS_HELP_STRING so the columns are automatically matched:

```bash
PHP_ARG_ENABLE([rtld-now],
  [whether to dlopen extensions with RTLD_NOW instead of RTLD_AZY],
  [AS_HELP_STRING([--enable-rtld-now],
    [Use dlopen with RTLD_NOW instead of RTLD_LAZY])],
  [no],
  [no])
```

Reply:
indeed, appllied